src/test/java/org/tarantool/jdbc/JdbcResultSetIT.java

```diff
@@ -220,6 +220,38 @@ public void testNullsSortingDesc() throws SQLException {
         assertFalse(resultSet.next());
     }
 
+    @Test
+    void testObjectWasNullColumn() throws SQLException {
```

Comment:
I would also check null / non-null binary string in a scalar field. SCALAR is only type we don't check here as I see.